ansible/configs/ocp-workshop/files/repos_template.3.9.14.j2

```diff
@@ -0,0 +1,43 @@
+[rhel-7-server-rpms]
+name=Red Hat Enterprise Linux 7
+baseurl={{own_repo_path}}/rhel-7-server-rpms
+enabled=1
+gpgcheck=0
+
+[rhel-7-server-rh-common-rpms]
+name=Red Hat Enterprise Linux 7 Common
+baseurl={{own_repo_path}}/rhel-7-server-rh-common-rpms
+enabled=1
+gpgcheck=0
+
+[rhel-7-server-extras-rpms]
+name=Red Hat Enterprise Linux 7 Extras
+baseurl={{own_repo_path}}/rhel-7-server-extras-rpms
+enabled=1
+gpgcheck=0
+
+[rhel-7-server-optional-rpms]
+name=Red Hat Enterprise Linux 7 Optional
+baseurl={{own_repo_path}}/rhel-7-server-optional-rpms
+enabled=1
+gpgcheck=0
+
+[rhel-7-server-ose-{{repo_version}}-rpms]
+name=Red Hat Enterprise Linux 7 OSE {{repo_version}}
+baseurl={{own_repo_path}}/rhel-7-server-ose-{{repo_version}}-rpms
+enabled=1
+gpgcheck=0
+
+## Required since OCP 3.5
+[rhel-7-fast-datapath-rpms]
+name=Red Hat Enterprise Linux Fast Datapath (RHEL 7 Server) (RPMs)
+baseurl={{own_repo_path}}/rhel-7-fast-datapath-rpms
+enabled=1
+gpgcheck=0
+
+## Required since OCP 3.9
```

Comment:
can we just add a `if osrelease|version_compare('>=', '3.9')` in the other existing template ?
i don't think repos_template.j2 is gonna be as complicated as hosts_templates.j2